lib/chef/config.rb

```diff
@@ -569,12 +569,8 @@ def self.env
       ENV
     end
 
-    def self.windows_home_path
```

Comment:
It doesn't seem to be used anywhere else but here as a helper/for testing.  It does appear in the rubydocs when I google for it.  How do we do deprecations?